backend/canisters/community/CHANGELOG.md

```diff
@@ -5,10 +5,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [unreleased]
 
+## [[2.0.824](https://github.com/open-chat-labs/open-chat/releases/tag/v2.0.824-community)] - 2023-08-25
```

Comment:
I think we should just leave 824 out of the changelog. It can't accurately be shown since it's a fork